src/PagerScroll.js

```diff
@@ -140,6 +140,7 @@ export default class PagerScroll<T: *> extends React.Component<
       navigationState,
       onSwipeStart,
       onSwipeEnd,
+      bouncesEnabled,
```

Comment:
Thanks for the PR. Can you rename `bouncesEnabled` to just `bounces` to be consistent with RN's ScrollView naming?

Reply:
Sure. `bounces` make sense.